src/Microsoft.AspNetCore.Mvc.Core/Authorization/AuthorizeFilter.cs

```diff
@@ -106,6 +107,11 @@ public AuthorizeFilter(string policy)
                 throw new ArgumentNullException(nameof(context));
             }
 
+            if (context.Result != null)
+            {
+                return;
```

Comment:
This won't ever happen. Setting `context.Result` short circuits in auth filters and will prevent any later ones from running. https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/Internal/ResourceInvoker.cs#L255